FlightReservation/components/formComponent.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from 'react';
+import {
+    StyleSheet,
+    View,
+    Text,
+    Image,
+    TextInput,
+    Pressable,
+} from 'react-native';
+
+function Form(props) {
+    const {validEmail,setValidEmail, validPassword, setValidPassword, changeForm, objValues, isLoginFormActive } = props;
+    const [isSeePassword, setSeePassword] = useState(true);
+
+    //const [validEmail, setValidEmail] = useState(true);
+    //const [validPassword, setValidPassword] = useState(true);
+
+    const changeSeePassword = () => {
+        setSeePassword(!isSeePassword);
+    };
+    const inputBorderColor = {
+        borderColorName: '#818181',
+        borderColorEmail: '#818181',
+        borderColorPassword: '#818181',
+    };
+    const [borderColor, setBorderColor] = useState(inputBorderColor);
+    const checkInputsIsNull = (e, property, propertyBorder) => {
+        const textInput = e.nativeEvent.text;
+        changeForm(property, textInput);
+        const color = !textInput ? '#818181' : '#5B6EF8';
+        setBorderColor({
+            ...borderColor,
+            [propertyBorder]: color,
+        });
+    };
+
+    return (
+        <View style={styles.containerForm}>
+            {!isLoginFormActive &&
+                <View>                    
+                <Text style={styles.inputHeader}>First Name</Text>
+                <TextInput
+                    style={[styles.inputStyle, { borderColor: borderColor.borderColorName }]}
+                    value={objValues.name}
+                    onChange={(e) =>
+                        checkInputsIsNull(e, 'name', 'borderColorName')
+                    }></TextInput>
+                </View>
+            }
+            <View style={{ flexDirection: 'row' }}>
+                <Text style={styles.inputHeader}>Email</Text>
+                <Text style={validEmail ? { color: "gray" } : styles.errorText}>*</Text>
+                {!validEmail && <Text style={styles.errorText}>Email in use. Use a different email</Text>}
```

Comment:
The titles and all the texts have to be inside a constant file and only import here.

Reply:
Done!